src/Razor/src/Microsoft.VisualStudio.RazorExtension/Snippets/SnippetService.cs

```diff
@@ -23,45 +24,53 @@ internal class SnippetService
     private readonly JoinableTaskFactory _joinableTaskFactory;
     private readonly IAsyncServiceProvider _serviceProvider;
     private readonly SnippetCache _snippetCache;
+    private readonly IAdvancedSettingsStorage _advancedSettingsStorage;
     private IVsExpansionManager? _vsExpansionManager;
 
     private readonly object _cacheGuard = new();
 
     private static readonly Guid s_CSharpLanguageId = new("694dd9b6-b865-4c5b-ad85-86356e9c88dc");
     private static readonly Guid s_HtmlLanguageId = new("9bbfd173-9770-47dc-b191-651b7ff493cd");
 
-    private static readonly Dictionary<Guid, ImmutableHashSet<string>> s_ignoredSnippets = new()
+    private static readonly Dictionary<Guid, ImmutableHashSet<string>> s_inboxSnippets = new()
```

Comment:
nit: Would a name like `defaultSnippets` or `builtInSnippets` perhaps be clearer? I get what you mean with the current naming, but my first thought went to email 😳

Reply:
I like `buildInSnippets` :) 

Inbox is just from my old windows days for inbox apps :P